src/main/scala/mesosphere/mesos/Availability.scala

```diff
@@ -0,0 +1,25 @@
+package mesosphere.mesos
+
+import java.util.concurrent.TimeUnit
+
+import org.apache.mesos.Protos.Offer
+
+object Availability {
+
+  def offerAvailable(offer: Offer, drainingTime: Long): Boolean = {
```

Comment:
Better if `drainingTime` is of `Duration` time or it name suggest it's in seconds.